x-pack/plugins/apm/public/components/shared/TransactionBreakdown/TransactionBreakdownKpiList.tsx

```diff
@@ -53,7 +53,7 @@ function KpiDescription({ name, color }: { name: string; color: string }) {
 
 function TransactionBreakdownKpiList({ kpis }: Props) {
   return (
-    <EuiFlexGrid>
+    <EuiFlexGrid style={{ height: 82 }}>
```

Comment:
This feels like a magic number. What is it based on?

Reply:
Did you happen to also check what happens if there are lots of KPIs and/or a smaller screen? Do the rows wrap? If so, will having a fixed `height` break this?

Reply:
> This feels like a magic number. What is it based on?

This is needed to keep the height of the container with the KPI list even when there is no data. 

> Did you happen to also check what happens if there are lots of KPIs and/or a smaller screen? Do the rows wrap? If so, will having a fixed height break this?

No, I haven't tested it, but I'm pretty sure that it'll break. Having that said, I'd rather remove the fixed height, and change the chart height when there is no data. The current height, when there is data, is `unit(16px) * 12`, and when there is no I can set it to `undefined` then the chart will use the default height size, which is the same for the other charts. WDYT?

Reply:
Agree we should be careful about hardcoding magic numbers (although they are sometimes necessary)

Reply:
The file was removed.